src/Sarif/Core/Location.cs

```diff
@@ -22,5 +22,10 @@ public LogicalLocation LogicalLocation
                 }
             }
         }
+
+        public bool ShouldSerializeId()
```

Comment:
>ShouldSerializeId [](http://example.com/codeflow?start=20&length=17)

Ah! you moved this out of the 'NotYetAutogenerated' directory, now I understand. But isn't it true that this isn't required because of the serialization attributes set on the Location 'Id' property? Or am I mistaken?

Reply:
I believe it is needed, please see the comment on the place I removed it.